prerequisites.md

```diff
@@ -60,30 +60,37 @@ The following packages are required, and used by the rest of these docs
 ```bash
 $ curl -sL https://deb.nodesource.com/setup_16.x | sudo -E bash -
 $ sudo apt-get update -y && sudo apt-get install -y \
+    autoconf \
+    automake \
+    autotools-dev \
     build-essential \
-    jq \
-    netcat \
-    nodejs \
+    bsdmainutils \
```

Comment:
this is also interesting - is this required for the changes in how BDB is built?

Reply:
Not required, removed it